<<<<<<< HEAD
<documentation title="Function Call Argument Spacing">
    <standard>
    <![CDATA[
    There should be no space before and exactly one space, or a new line, after a comma when passing arguments to a function or method.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: No space before and exactly one space after a comma.">
        <![CDATA[
foo($bar,<em> </em>$baz);
        ]]>
        </code>
        <code title="Invalid: A space before and no space after a comma.">
        <![CDATA[
foo($bar<em> </em>,<em></em>$baz);
=======
<documentation title="Function Argument Spacing">
    <standard>
    <![CDATA[
    Function arguments should have one space after a comma, and single spaces surrounding the equals sign for default values.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Single spaces after a comma.">
        <![CDATA[
function foo($bar,<em> </em>$baz)
{
}
        ]]>
        </code>
        <code title="Invalid: No spaces after a comma.">
        <![CDATA[
function foo($bar,<em></em>$baz)
{
}
        ]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: Single spaces around an equals sign in function declaration.">
        <![CDATA[
function foo($bar, $baz<em> </em>=<em> </em>true)
{
}
        ]]>
        </code>
        <code title="Invalid: No spaces around an equals sign in function declaration.">
        <![CDATA[
function foo($bar, $baz<em></em>=<em></em>true)
{
}
>>>>>>> ddb2375 (fix: console error)
        ]]>
        </code>
    </code_comparison>
</documentation>
